react/Button/index.jsx

```diff
@@ -43,13 +43,15 @@ export const Button = props => {
 export default Button
 
 export const ButtonLink = props => {
-  const { theme, size, extension, href, target, className, children, label, icon, onClick } = props
+  const { disabled, theme, size, extension, href, target, className, children, label, icon, onClick } = props
   return (
     <a
+      aria-disabled={disabled}
+      disabled={disabled}
```

Comment:
The disabled attribute is not valid for anchors. https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes